[2024-01-21T11:35:18.573-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_sql_pipeline.update_values manual__2024-01-21T19:35:07.934519+00:00 [queued]>
[2024-01-21T11:35:18.583-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_sql_pipeline.update_values manual__2024-01-21T19:35:07.934519+00:00 [queued]>
[2024-01-21T11:35:18.583-0800] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:35:18.583-0800] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-01-21T11:35:18.583-0800] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:35:18.607-0800] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): update_values> on 2024-01-21 19:35:07.934519+00:00
[2024-01-21T11:35:18.611-0800] {standard_task_runner.py:55} INFO - Started process 30640 to run task
[2024-01-21T11:35:18.615-0800] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_sql_pipeline', 'update_values', 'manual__2024-01-21T19:35:07.934519+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/executing_sql_pipeline.py', '--cfg-path', '/tmp/tmpv_4kho7l']
[2024-01-21T11:35:18.616-0800] {standard_task_runner.py:83} INFO - Job 87: Subtask update_values
[2024-01-21T11:35:18.673-0800] {task_command.py:389} INFO - Running <TaskInstance: executing_sql_pipeline.update_values manual__2024-01-21T19:35:07.934519+00:00 [running]> on host Mittu.
[2024-01-21T11:35:18.736-0800] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=madhu
AIRFLOW_CTX_DAG_ID=executing_sql_pipeline
AIRFLOW_CTX_TASK_ID=update_values
AIRFLOW_CTX_EXECUTION_DATE=2024-01-21T19:35:07.934519+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-21T19:35:07.934519+00:00
[2024-01-21T11:35:18.736-0800] {sql.py:254} INFO - Executing: 
            UPDATE users SET city = 'Seattle';
        
[2024-01-21T11:35:18.740-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:35:18.820-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:35:18.821-0800] {sql.py:375} INFO - Running statement: 
            UPDATE users SET city = 'Seattle';
        , parameters: None
[2024-01-21T11:35:18.821-0800] {sql.py:384} INFO - Rows affected: 6
[2024-01-21T11:35:18.838-0800] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=executing_sql_pipeline, task_id=update_values, execution_date=20240121T193507, start_date=20240121T193518, end_date=20240121T193518
[2024-01-21T11:35:18.867-0800] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-21T11:35:18.881-0800] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
